README.md

```diff
@@ -44,12 +46,20 @@ scons platform=iphone target=debug
 
 ### For Godot 4.x:
 
-```bash
-scons platform=ios target=debug
-```
 
-You don't have to wait for full engine compilation, as header files are generated first.
-Once the actual compilation starts, you can stop this command by pressing <kbd>Ctrl + C</kbd>.
+1. After cloning this repository, go to `godot` submodule and checkout the tag of the engine version you are working with. For example `4.3-stable`.

```

Comment:
`git switch` is preferred over `git checkout` nowadays, as it avoids the ambiguity of `git checkout` also being able to work as `git restore`:

```suggestion
1. After cloning this repository, go to `godot` submodule and switch to the tag of the engine version you are working with. For example, `git switch 4.4-stable`.

```

Reply:
It's a tag so should be "git checkout" and not "switch".